### ISSUE TITLE
Feature request - Lost Cities triggers

### ISSUE THREAD

**Issue by anhgelus** [TheComputerizer/Music-Triggers#128]
**Describe your idea**
Add triggers related to our position in cities (in building, in street...) generated by the mod [The Lost Cities](https://modrinth.com/mod/the-lost-cities).

**Describe alternatives you've considered**
Not applicable.

**Comment by TheComputerizer** [comment]
Is there a specific need to do this when the dimension, structure, height, and zone triggers already exist?

**Comment by anhgelus** [comment]
Yes because this mod is modifying the default generation of the overworld by generating random cities without using structures.
We can't use the height and zone triggers because the cities are generated randomly (defining a zone for each city for each map is not possible).
BUT the dimension added by Lost Cities can be handled by the dimension trigger.

**Comment by TheComputerizer** [comment]
I will keep it in mind and take a look for 6.4 since 6.3 is almost released

**Comment by TheComputerizer** [comment]
This is on the radar for 7.1 after 7.0 is stable
